rgf/rgf_model.py

```diff
@@ -381,6 +381,38 @@ def _find_model_file(self):
                             'Training is abnormally finished.'.format(utils.TEMP_PATH))
         self._model_file = sorted(model_files, reverse=True)[0]
 
+    def dump_model(self):
```

Comment:
Dump is not wrong since it includes the meaning of "print".
Should I change it to `print_model`?

Or leave it to prepare for `print_to_console` argument?